spec/simple.spec.ts

```diff
@@ -199,6 +199,63 @@ describe('Simple', () => {
     });
   });
 
+  it('supports native string enums', () => {
+    enum NativeEnum {
+      OPTION = 'a',
+      ANOTHER = 'b',
+      DEFAULT = 'c',
+    }
+
+    const nativeEnumSchema = z.nativeEnum(NativeEnum).openapi({
+      refId: 'NativeEnum',
+      description: 'A native string enum in zod',
+    });
+
+    expectSchema([nativeEnumSchema], {
+      NativeEnum: {
+        type: 'string',
+        description: 'A native string enum in zod',
+        enum: ['a', 'b', 'c'],
+      },
+    });
+  });
+
+  it('does not support mixed native enums', () => {
```

Comment:
Maybe a describe for enums could help here. Not sure if you already have this in the other PR

Reply:
I have that in the other PR